### DISCUSSION TITLE
Naming of TwitchPlays

### DISCUSSION THREAD

**Discussion by howroyd** [howroyd/simonsays#20] [Polls]
Feel free to add suggestions either via #6 or by jumping in chat now on [Twitch](https://www.twitch.tv/drgreengiant)!

Main rules is the name must be unique; i.e. do a Google search and make sure nothing else uses that name already.

Must not contain existing trade/brand/product names like "Twitch," for example (unless the rights are owned by me.)

**Comment by howroyd** [comment]
Extra suggestions.  Just comment below if you prefer them to one of the ones in the poll above:

```
Simon Says
GreenGiantTrolling
```

**Reply by BernadetteBotBasher** [reply]
Yess, GreenGiantTrolling

**Comment by Szinie** [comment]
Ok I voted sth else but Simon Says is actually brilliant!
